pkg/intcode/intcode.go

```diff
@@ -103,7 +79,7 @@ func (p *program) extractArguments() (op opcode, arg1, arg2, arg3 int) {
 	)
 
 	op = opcode(instruction % 100)
-	if op > p.maxOp {
+	if op > p.maxOp || op == 0 {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [avoid magic numbers like '0', create a named constant for it](https://app.codacy.com/manual/afarbos/aoc/pullRequest?prid=5498712)